src/status_im/contexts/wallet/events.cljs

```diff
@@ -41,18 +41,18 @@
 (rf/reg-event-fx :wallet/navigate-to-account
  (fn [{:keys [db]} [address]]
    {:db (assoc-in db [:wallet :current-viewing-account-address] address)
-    :fx [[:dispatch [:navigate-to :screen/wallet.accounts address]]]}))
+    :fx [[:dispatch [:navigate-to :screen/wallet.accounts]]]}))
 
 (rf/reg-event-fx :wallet/navigate-to-account-within-stack
  (fn [{:keys [db]} [address]]
    {:db (assoc-in db [:wallet :current-viewing-account-address] address)
-    :fx [[:dispatch [:navigate-to-within-stack [:screen/wallet.accounts :shell-stack] address]]]}))
+    :fx [[:dispatch [:navigate-to-within-stack [:screen/wallet.accounts :shell-stack]]]]}))
 
 (rf/reg-event-fx :wallet/navigate-to-new-account
  (fn [{:keys [db]} [address]]
    {:db (assoc-in db [:wallet :current-viewing-account-address] address)
     :fx [[:dispatch [:hide-bottom-sheet]]
-         [:dispatch [:navigate-to :screen/wallet.accounts address]]
+         [:dispatch [:navigate-to :screen/wallet.accounts]]
```

Comment:
Minor cleanup: When the `:screen/wallet.accounts` was first created. it received the address as navigation params and displayed the account details (tokens, collectibles and other data accordingly). But, later we moved to store the address in the `:current-viewing-account-address` key in `app-db`.